front/src/styles/scss/applications/stdcmV2/_home.scss

```diff
@@ -1,12 +1,33 @@
 .stdcm-v2 {
+  /* 
+  We set the cursor to default to avoid the pointer cursor that appears due to role="button" on the div.
+  The role="button" makes the div appear clickable, which causes the cursor to change to a pointer on hover.
+  Using !important ensures that the cursor remains in its default state.
+*/
+  cursor: default !important;
```

Comment:
Sorry you had to do that :)

Reply:
Don't worry , it much better than before like this.